setup.py

```diff
@@ -22,6 +22,22 @@
             "ccx = lms.djangoapps.ccx.plugins:CcxCourseViewType",
             "edxnotes = lms.djangoapps.edxnotes.plugins:EdxNotesCourseViewType",
             "instructor = lms.djangoapps.instructor.views.instructor_dashboard:InstructorDashboardViewType",
+            "courseware = openedx.core.djangoapps.content.course_views.tabs:CoursewareTab",
```

Comment:
One thing that was next on my list was moving all of these to be `CourseViewType` subclasses. I just hadn't gotten around to it yet.